Tests/SwiftDocCTests/Infrastructure/AutomaticCurationTests.swift

```diff
@@ -448,7 +448,40 @@ class AutomaticCurationTests: XCTestCase {
             )
         }
     }
-    
+
+    func testNoAutoCuratedMixedLanguageDuplicates() throws {
+        // The symbol graphs for MixedLanguageFramework.docc declare a protocol called MixedLanguageProtocol.
+        // This test checks that there is only one child node under "MixedLanguageProtocol-Implementations" -> "Instance Methods",
+        // even though there are two languages and two symbol graph files.
+        //   doc://org.swift.MixedLanguageFramework/documentation/MixedLanguageFramework
+        //   ├ doc://org.swift.MixedLanguageFramework/documentation/MixedLanguageFramework/MixedLanguageClassConformingToProtocol
+        //   │ ├ doc://org.swift.MixedLanguageFramework/documentation/MixedLanguageFramework/MixedLanguageClassConformingToProtocol/MixedLanguageProtocol-Implementations
+        let (bundle, context) = try testBundleAndContext(named: "MixedLanguageFramework")
+
+        let protocolImplementationsNode = try context.entity(
+            with: ResolvedTopicReference(
+                bundleIdentifier: bundle.identifier,
+                path: "/documentation/MixedLanguageFramework/MixedLanguageClassConformingToProtocol/MixedLanguageProtocol-Implementations",
+                sourceLanguages: [.swift, .objectiveC]
+            )
+        )
+
+        // This page should contain an auto-curated "Instance Methods" task group.
+        let protocolImplementationsArticle = protocolImplementationsNode.semantic as! Article
+        XCTAssertEqual(1, protocolImplementationsArticle.automaticTaskGroups.count)
+        let instanceMethodsTaskGroup = protocolImplementationsArticle.automaticTaskGroups.first!
```

Comment:
minor: We try to use `try XCTUnwrap(...)` instead of force unwrapping in tests. The difference is that if something unexpectedly breaks in the future, that surfaces as a test failure—which lets other tests run as well—instead of a crash—which aborts other test and don't give a complete report of how many tests passes or failed.